submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/handlers/SubmitWebHandler.kt

```diff
@@ -86,6 +115,11 @@ class SubmitWebHandler(
                     serializationService.deserializeSubmission(rqt.submission, rqt.format, source)
 
                 is FileSubmitWebRequest -> serializationService.deserializeSubmission(rqt.submission, source)
+
+                is DraftSubmitWebRequest -> {
+                    val draft = requestDraftService.getRequestDraft(rqt.key, rqt.owner)
```

Comment:
I could try this. I just wanted to have a distinction from when we're submitting with draft rather than a regular submission but we're currently not storing this anywhere and maybe it was just an abstraction I missed. This was a long PR 🫠